website/website/add_youtuber.py

```diff
@@ -0,0 +1,77 @@
+from googleapiclient.discovery import build
+from urllib.parse import urlparse, unquote
+import os
+API_KEY = os.environ.get('API_KEY')
+
+
+class YoutubeApi:
+    """A class to interact with the YouTube API."""
+
+    def __init__(self, API_KEY, channel_url) -> None:
+        self.API_KEY = API_KEY
+        self.channel_url = channel_url
+        self.username = None
+        self.channel_id = None
+        self.channel_title = None
+        self.channel_description = None
```

Comment:
I'd consider declaring all the possible class attributes on the class level.

So, it can be documented and you can easily see all the class possible attributes in the same place.

e.g.:

class YoutubeApi:
    #: str: The identifier of the channel user, who owns the channel.
    username = None
    #: str: The identifier of the channel.
    channel_id = None

    ....